src/transformers/models/longformer/modeling_tf_longformer.py

```diff
@@ -2079,10 +2079,12 @@ def get_prefix_bias_name(self):
     @add_start_docstrings_to_model_forward(LONGFORMER_INPUTS_DOCSTRING.format("batch_size, sequence_length"))
     @add_code_sample_docstrings(
         processor_class=_TOKENIZER_FOR_DOC,
-        checkpoint=_CHECKPOINT_FOR_DOC,
+        checkpoint="saibo/legal-longformer-base-4096",
```

Comment:
Is any particular reason not to use the TF checkpoint from 
allenai/longformer-base-4096
here?

Reply:
no reasons apologies didn't realise that checkpoint had a TF one. Switched to that now.